e2e/e2e.go

```diff
@@ -314,6 +314,13 @@ func shouldRunE2ETests() (bool, error) {
 			return err
 		}
 
+		if info.IsDir() && path != "." {
```

Comment:
Would it make more sense to see if it's a valid submodule by checking `.gitmodules` in the root directory?

Reply:
maybe